packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -48,10 +49,12 @@ export async function createSignSendProcessXCPDRequest({
     try {
       await axios.post(pdResponseUrl, result);
     } catch (error) {
+      const errorString = errorToString(error);
       capture.error("Failed to send PD response to Internal Carequality Endpoint", {
         extra: {
-          error,
+          errorString,
           result,
+          xcpdRequest,
```

Comment:
including the request and condensing the error for easier debugging

Reply:
We still want to capture these since these are an immediate issue that need to be remediated if they occur. They are an error in our system and not in an external gateways system